drivers/filesystems/udfs/env_spec.cpp

```diff
@@ -308,7 +308,7 @@ UDFPhWriteSynchronous(
 #endif //MEASURE_IO_PERFORMANCE
 
 #if defined UDF_DBG || defined USE_PERF_PRINT
-    ULONG Lba = (ULONG)(Offset>>0xb);
+//    ULONG Lba = (ULONG)(Offset>>0xb);
```

Comment:
That's inconsistent, because now what would happen if one re-enables the NDEBUG or whatever? The PerfPrint call below needs that Lba variable.